src/components/RegisterProgram/components/Content/DetailOrder.tsx

```diff
@@ -102,42 +130,65 @@ const Order: FC<DataItem> = ({ qty = 0, totalPrice, data }) => {
   )
 }
 
-// TODO: Get from user voucher
-const Voucher: FC<DataItem> = ({ totalPrice }) => {
+const Voucher: FC<VoucherProps> = ({ loading, voucherBalance, totalPrice }) => {
+  const remainingVoucher = useMemo(() => {
+    const priceValue = totalPrice || 0
+
+    if (voucherBalance - priceValue < 0) {
+      return priceValue - voucherBalance
+    } else {
+      return voucherBalance - priceValue
+    }
```

Comment:
```suggestion
     return Math.abs(voucherBalance - priceValue);
```
bisa gini nihh setau aku, cobaindeh siapatau bisa